packages/documentation/src/elements/navigation-rail/navigation-rail-item.mdx

```diff
@@ -0,0 +1,13 @@
+import { Meta, Canvas } from "@storybook/blocks";
+import * as Stories from "./navigation-rail-item.stories.ts";
+
+<Meta of={Stories} />
+
+# Navigation Rail Items
```

Comment:
Für das Navigation rail item würde ich keine separate story aufnehmen, eher eine Sektion in der haupt-story.